src/UrqlClient.re

```diff
@@ -260,6 +264,8 @@ type clientOptions('a, 'b) = {
   suspense: bool,
   [@bs.optional]
   fetch: 'b,
+  [@bs.optional]
+  requestPolicy: UrqlTypes.requestPolicy,
```

Comment:
Nice! We'll just need to make sure this gets exposed in a few more places. This `[@bs.deriving abstract]` binds the JS API, but we need to make sure the Reason API has a way for us to send this value along. Here are the steps we'll need to take:

1. Change this line to:

```suggestion
  [@bs.optional]
  requestPolicy: string,
```

In JS-land, this will be a `string`. The polymorphic variant we have for `requestPolicy` guarantees it'll never be a string other than those specified here: https://github.com/FormidableLabs/reason-urql/blob/master/src/UrqlTypes.re#L3-L9

2. Add an argument for `requestPolicy` here: https://github.com/FormidableLabs/reason-urql/blob/master/src/UrqlClient.re#L280, i.e. ~requestPolicy=`CacheFirst (sorry I can't code highlight here, the backtick for polymorphic variants is important). 
3. Convert from the polymorphic variant to the string representation here: https://github.com/FormidableLabs/reason-urql/blob/master/src/UrqlClient.re#L290 by adding: `~requestPolicy=requestPolicy->UrqlTypes.requestPolicyToJs`

This last piece ensures when we receive the polymorphic variant in Reason we turn it into a string on the JS-side.